pkgs/development/python-modules/mpv-jsonipc/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildPythonPackage, fetchFromGitHub, requests
+, tqdm, websocket_client, pytest, pillow, mock, isPy27 }:
+
+buildPythonPackage rec {
+  pname = "mpv-jsonipc";
+  version = "unstable-2020-02-01";
```

Comment:
Please explain why https://github.com/pkkid/python-plexapi/releases/tag/3.2.0 isn't new enough. You may want to request upstream to release a new version...

Reply:
I'm confused. What does `python-plexapi` have to do with `mpv-jsonipc`? They're two different libraries with different purposes?

Reply:
I finally realized the source of confusion, I had left the metadata at the bottom from the python-plexapi package and hadn't noticed until just a few minutes ago. This is resolved as a result.